drivers/SmartThings/zwave-thermostat/src/ct100-thermostat/init.lua

```diff
@@ -258,40 +117,40 @@ local function thermostat_mode_report_handler(self, device, cmd)
   end
 end
 
-local function temperature_report_handler(self, device, cmd)
-  if (cmd.args.sensor_type == SensorMultilevel.sensor_type.TEMPERATURE) then
-    local scale = 'C'
-    if (cmd.args.scale == SensorMultilevel.scale.temperature.FAHRENHEIT) then scale = 'F' end
-    device:emit_event_for_endpoint(cmd.src_channel, capabilities.temperatureMeasurement.temperature({value = cmd.args.sensor_value, unit = scale}))
-    device:set_field(TEMPERATURE, cmd.args.sensor_value, {persist = true})
-  end
+local function do_refresh(self, device, cmd)
```

Comment:
nit: cmd is unused

Reply:
@L153 it's set as the capability command handler

Reply:
and called on L132

Reply:
I meant the argument `cmd` is not used in the function, the only usage also passes in nil...

Reply:
ooooh